src/main/resources/database/changelog/02-add-enable-daily-notifications.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
```

Comment:
I do not understand, why we need new file for this fix. Can you clarify please?

Reply:
Every change in Luquibase must be incremental.
We already have a database running in Heroku Postgres, which scheme was defined by a previous changeset. Thus, we can't edit that one and must create a new one instead.
Changesets can be either listed in separate files or in a single file (or mixed). I personally find having each changeset in a separate file more convenient to track changes.